src/org/jitsi/impl/neomedia/MediaStreamImpl.java

```diff
@@ -3751,12 +3752,47 @@ public int getSpatialID(byte[] buf, int off, int len)
         }
         else
         {
-            logger.warn("Method not implemented");
+            // XXX not implementing temporal layer detection should not break
+            // things.
             return -1;
         }
     }
 
     /**
+     * Returns a boolean that indicates whether or not our we're able to detect
+     * the frame boundaries for the codec of the packet that is specified as an
+     * argument.
+     *
+     * @param baf the {@link ByteArrayBuffer} that holds the RTP packet.
+     *
+     * @return true if we're able to detect the frame boundaries for the codec
+     * of the packet that is specified as an argument, false otherwise.
+     */
+    public boolean supportsFrameBoundaries(ByteArrayBuffer baf)
+    {
+        if (frameMarkingsExtensionId == -1)
+        {
+            REDBlock redBlock = getPayloadBlock(baf);
+            if (redBlock != null && redBlock.getLength() != 0)
+            {
+                final byte vp9PT = getDynamicRTPPayloadType(Constants.VP9),
+                    vp8PT = getDynamicRTPPayloadType(Constants.VP8),
+                    pt = redBlock.getPayloadType();
+
+                return vp9PT == pt || vp8PT == pt;
```

Comment:
should there be some h264 support here?  or you're thinking that will have to fall under the frameMarkingExtension scenario?  could't timestamp be used for h264?  or maybe this is only given a single packet.